### PR TITLE
FastDeserializerGenerator skipBytes() semicolon typo fix

### PR DESCRIPTION
**Pull request by flowenol**
Looks like a simple typo, but apparently this hasn't been covered by the tests so far. Besides simple fix I have also refactored the name of the test case from the previous PR and extended it to cover the issue with `skipBytes()`.

### CODE REVIEW

**Review by radai-rosenblatt — APPROVED** [review]

File: avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/deserialization/AVRO_1_4/ClasspathTestRecord_SpecificDeserializer_4580844570251215039_2703298965059642600.java [hunk 1]
```diff
@@ -1,269 +0,0 @@
...
-package com.linkedin.avro.fastserde.generated.deserialization.AVRO_1_4;
-
-import java.io.IOException;
-import com.linkedin.avro.fastserde.FastDeserializer;
-import org.apache.avro.Schema;
-import org.apache.avro.generic.GenericArray;
-import org.apache.avro.io.Decoder;
-import org.apache.avro.util.Utf8;
-
-public class ClasspathTestRecord_SpecificDeserializer_4580844570251215039_2703298965059642600
```
Comment (FelixGV) [inline avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/deserialization/AVRO_1_4/ClasspathTestRecord_SpecificDeserializer_4580844570251215039_2703298965059642600.java:11]: It seems like we're replacing this code-generated class with another one by altering the test. Can we check-in the replacement for this class as part of this first commit? The goal is to be able to easily visualize the diff to the codegen output in the third commit.

Comment (flowenol) [reply avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/deserialization/AVRO_1_4/ClasspathTestRecord_SpecificDeserializer_4580844570251215039_2703298965059642600.java:11]: Yes, I decided to change the test name to make it more appropriate for the actual case. Please, check the latest push. The first commit contains the refactored test with codegen classes, which don't compile due to the semicolon issue. The second commit is the simple fix, and the last one are the fixed codegen classes.

**Comment by flowenol** [comment]
@FelixGV @radai-rosenblatt  is there anything else blocking this pull request?

**Review by FelixGV — APPROVED** [review]
Thanks!

**Comment by FelixGV** [comment]
FYI, this commit is included in release 0.2.11... thanks for your patience!

**Pull request merged**